src/components/colorPicker.tsx

```diff
@@ -18,7 +20,11 @@ const ColorPicker = ({
 				key={colorItem}
 			>
 				<div
-					className="h-4 w-4 rounded-full border-[1px] border-gray-900 p-1"
+					className={`h-4 w-4 rounded-full border-[1px] p-1 ${
+						colorItem === colorPickerColors[0]
```

Comment:
Can we try and use classnames here https://www.npmjs.com/package/classnames ? this line will be much cleaner